nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1517,12 +1513,16 @@ private void checkFieldInitialization(ClassTree tree, VisitorState state) {
       }
     }
     for (Element constructorElement : errorFieldsForInitializer.keySet()) {
-      List<Symbol> fieldSymbols =
-          errorFieldsForInitializer
-              .get(constructorElement)
-              .stream()
-              .map(element -> ASTHelpers.getSymbol(getTreesInstance(state).getTree(element)))
-              .collect(Collectors.toList());
+      ImmutableList<Symbol> fieldSymbols =
+          ImmutableList.<Symbol>builder()
+              .addAll(
+                  errorFieldsForInitializer
+                      .get(constructorElement)
+                      .stream()
+                      .map(
+                          element -> ASTHelpers.getSymbol(getTreesInstance(state).getTree(element)))
+                      .collect(Collectors.toList()))
+              .build();
```

Comment:
I think I said this before, but `ImmutableList` also has a collector, so no need to generate the intermediate mutable list and copy the elements. See my response comment to the original change here.